x11-terms/xfce4-terminal/xfce4-terminal-1.0.4-r1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="A terminal emulator for the Xfce desktop environment"
+HOMEPAGE="https://docs.xfce.org/apps/terminal/start"
+SRC_URI="https://archive.xfce.org/src/apps/${PN}/$(ver_cut 1-2)/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-solaris"
+IUSE="utempter"
+
+RDEPEND="
+	>=dev-libs/glib-2.42:2=
+	>=x11-libs/gtk+-3.22:3=
+	x11-libs/libX11:=
+	>=x11-libs/vte-0.51.3:2.91=
+	>=xfce-base/libxfce4ui-4.19:=[gtk3(+)]
+	>=xfce-base/xfconf-4.16:=
+	utempter? ( sys-libs/libutempter:= )
+"
+DEPEND="
+	${RDEPEND}
+"
+BDEPEND="
+	dev-libs/libxml2
+	dev-util/intltool
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	subst_args=(xfce_gtk_menu_append_sep{e,a}rator)
+	sed -E -i "s#${subst_args[0]}#${subst_args[1]}#" terminal/terminal-window{,-dropdown}.c || die
```

Comment:
This is the kind of thing which is _really_ better for both:
1. a patch (... we don't want to apply it anymore once it's fixed, and it's a precise change);
2. a patch which then goes upstream.

Reply:
already upstream: https://github.com/xfce-mirror/xfce4-terminal/commit/d9e909946951fff0c216e3338df5adfbf9b2aa1f
i think this is just a like a "transition" ebuild fix.
we could also just wait…

Reply:
or introduce backword compatiblity in libxfce4ui by reverting/reverse applying
--> https://gitlab.xfce.org/xfce/libxfce4ui/-/merge_requests/88/diffs?commit_id=08076f487296a7a9673ad7e8547a48ba41beb74f